app/Console/Commands/ObjectImportCommand.php

```diff
@@ -170,12 +171,22 @@ public function fire()
 
                 $item["user"] = $this->createOrFetchUser($row, $password);
 
-                $item["location"] = $this->createOrFetchLocation($item_location);
-                $item["category"] = $this->createOrFetchCategory($item_category, $item_type);
-                $item["manufacturer"] = $this->createOrFetchManufacturer($row);
-                $item["company"] = $this->createOrFetchCompany($item_company_name);
+                if (!($this->updating && empty($item_location))) {
```

Comment:
Should this be `empty($item_location))` or `!empty($item_location))`? We only want to update it if there is a value, right?